docs/src/pages/components/buttons/buttons.md

```diff
@@ -180,3 +180,46 @@ However:
 ```
 
 This has the advantage of supporting any element, for instance, a link `<a>` element.
+
+## Unstyled button
```

Comment:
I'd say it's simpler to have it without the component name.

Reply:
I have asked because we have done it in two different ways:

- https://next.material-ui.com/components/modal/#unstyled
- https://next.material-ui.com/components/switches/#unstyled-switches